<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getWindow</class>
 <widget class="QMainWindow" name="getWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Выберите период:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>40</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="secondDayBox">
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>31</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="secondMonthBox">
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="secondYearBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>2021</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2022</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2023</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2025</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Вывести</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="firstDayBox">
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>31</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="firstMonthBox">
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="firstYearBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>2021</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2022</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2023</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2025</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
